core/crypto/client_impl.go

```diff
@@ -44,31 +46,39 @@ func (client *clientImpl) NewChaincodeDeployTransaction(chaincodeDeploymentSpec
 	}
 
 	// Get next available (not yet used) transaction certificate
-	tCert, err := client.tCertPool.GetNextTCert()
+	tCerts, err := client.tCertPool.GetNextTCerts(1)
 	if err != nil {
 		client.error("Failed getting next transaction certificate [%s].", err.Error())
 		return nil, err
 	}
+	if len(tCerts) != 1 {
+		client.error("Failed obtaining a TCert for Chaincode execution [%s].", err.Error())
```

Comment:
This needs to read "Failed obtaining a TCert for Chaincode deployment [%s]" to be consistent with the other invoke/query messages.